docs/GettingStarted/configuration/ExporterCommittime.md

```diff
@@ -1,9 +1,18 @@
 # Commit Time Exporter
 
 The job of the commit time exporter is to find and associate time of the relevant source code commit with a container image SHA built from that source code.
+
+This information can be found in two ways:
+
+- [git](#using-commit-time-with-git-apis) making a request to the Git provider
+
+- [image](#using-commit-time-with-images) querying `Annotations` or `Labels` within the Image object, without reaching to the git external services. For the image Time provider, the Commit Date must be present as a valid string. This allows to give greater flexibility around 3rd party CI systems used to create container images for the OpenShift running applications.
+
+
 Later the Deploy Time Exporter can associate that image SHA with a production deployment and allow to calculate [Lead Time for Change](../dashboards/SoftwareDeliveryPerformance/#lead-time-for-change) metrics.
 
-Commit Time Exporter may be used with an Build or Image cluster objects.
+> *NOTE:* It is important to deploy a `committime` exporter instance per unique Source Control provider. So if your monitored applications uses container images that are built from the source code hosted in the GitHub, Gitea, and two different self hosted BitBucket systems, you would need to create four committime instances.
+
 
 ## Using Commit Time with Git APIs
```

Comment:
I would put the Annotations sections from Overview here

Reply:
done